tinygrad/mlops.py

```diff
@@ -55,6 +55,18 @@ def backward(self, grad_output):
     return max_is_amount.binary_op(BinaryOps.MUL, grad_output_expanded)
 
 # ************* binary ops *************
+class CompareLess(Function):
```

Comment:
Need backward pass :)

Reply:
yeah, been trying to find more info on what a backwards pass for a comparison function does. would it just return the grad_output directly?

Reply:
grad_output masked where it was true i think. make tests to compare to pytorch

Reply:
added the tests to compare to pytorch, using forward_only as it will throw a runtime error due to missing grad_fn

should we match pytorch here by throwing an error for the comparison ops on the backwards pass?

Reply:
ended up throwing a runtime error to match pytorch